Loading.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
This file was also combined into `Gaming.html`, and not seemed to be changed.
So, this file should be deleted.

Reply:
ok I see!